### DISCUSSION TITLE
LessonNine - Help needed on random guess challenge

### DISCUSSION THREAD

**Discussion by vengateshs** [Cyfrin/foundry-full-course-cu#1187] [General]
Could not make out from the existing discussions on this challenge.

Any new hint is highly appreciated.

Thanks!

**Comment by xyz899** [comment]
Hum, Maybe explain the problem and paste the code if you want me to solve it ?
I'm not a superhero, can't solve something out of thin air.

**Reply by vengateshs** [reply]
In another thread, Patrick commented there is a way to solve this challenge without deploying.

Was just curious to know if someone has any idea about this approach. If so, any possible hint!!
